packages/template-cli/README.md

```diff
@@ -26,67 +26,111 @@ Creates an amidostacks template in your local directory much like express-genera
+3. Change to a directory for testing - e.g. $HOME and run
+```bash
+$ cd ${SELECTED_DIR}
+$ npx ${YOUR_GIT_STACKS_WEB_APP_PATH}/stacks-webapp-template/packages/template-cli/dist/index.js run -i
+```
 
-4. Navigate to the created project directory to install template dependencies:
+4. Follow the CLI prompts to create a templated app with user defined configuration:
+   - Project Name: _default_: current directory (will create a directory with this name)
+   - Project Type: _default_: SSR (server side rendering)
+   - Platform: _default_: AKS (Azure Kubernetes Service)
+   - CI/CD Tooling: _default_: AzureDevOps
+
+5. Navigate to the created project directory to install template dependencies:
 e.g. ${SELECTED_DIR}/$PROJECT_NAME
 ```bash
-cd ${SELECTED_DIR}/$PROJECT_NAME/src/ssr
-npm install
+$ cd ${SELECTED_DIR}/$PROJECT_NAME/src/
+$ npm install
 ```
 
-5. Build and run the app:
+6. Build and run the app:
 ```bash
-npm run build
-npm run start
+$ npm run build
+$ npm run start
 ```
 Open Browser and hit [http://localhost:3000](http://localhost:3000)
 
-6. To test the deploy folder has been correctly provisioned prior to checking you need to at this point in time copy over a sample backend-config and terraform vars. Currently vars.tf and provider configuration is not automatically updated. 
+7. To get a local deployment open the following files and uncomment these lines
+
+`${YOUR_GIT_STACKS_WEB_APP_PATH}/stacks-webapp-template/deploy/terraform/azure/provider.tf`
+```terraform
+   # resource_group_name  = "amido-stacks-rg-uks"
+   # storage_account_name = "amidostackstfstategbl"
+   # container_name       = "tfstate"
+   # # key                  = "spa-cycle-2"
+```
+
+`${YOUR_GIT_STACKS_WEB_APP_PATH}/stacks-webapp-template/deploy/terraform/azure/vars.tf`
+```terraform
+
+# locals {
+#   # // common //
+#   resource_prefix           = "${var.name_company}-${var.name_project}-${var.name_component}"
+#   resource_suffix           = "${var.location_name_map[var.resource_location]}-${var.name_environment}"
+
+#   # // resource group //
+#   resource_group_name       = "${local.resource_prefix}-rg-${local.resource_suffix}"
+
+#   # // storage account - static website hosting //
+#   storage_account_name      = "${var.name_company}${var.name_project}${var.name_component}sa${var.location_name_map[var.resource_location]}${var.name_environment}"
+# }
+```
+
+8. To test the deploy folder has been correctly provisioned prior to checking you need to at this point in time copy over a sample backend-config and terraform vars. Currently vars.tf and provider configuration is not automatically updated. 
 Future iterations will include this.
 
 The safest way to run and maintain this going forward is to rely on environment variables for credentials as that is the way the pipeline will trigger the executions of terraform. 
 
 Sample export script with correct environment vars:
 
 ```bash
-export ARM_CLIENT_ID= \
+$ export ARM_CLIENT_ID= \
```

Comment:
Do we need to include `set` here to for windows?

Reply:
I'm sure we do but I've never used powershell before and I don't own a windows machine. I'll have a play around with the docker powershell image and see what I can find

Reply:
will be done in next push